documents/motions/09.2022.1 - Senior Delegates.md

```diff
@@ -16,18 +16,19 @@ Senior Delegates are WCA Staff and Full Delegates, with an additional role to ma
    2. Senior Delegates can promote Trainee Delegates to Junior Delegates in their attributed area, provided they have attended at least one competition in the Trainee Delegate capacity.
    3. Senior Delegates can promote Junior Delegates to Full Delegates in their attributed area with approval from the WCA Board.
    4. Senior Delegates can appoint Delegates as Regional Delegates in their attributed area with approval from the WCA Board.
-   5. Senior Delegates can remove the role of WCA Delegate from Junior Delegates and Full Delegates in their attributed area with approval from the WCA Board.
+   5. Senior Delegates can remove the role of WCA Delegate from Junior Delegates and Full Delegates in their attributed area.
    6. Senior Delegates can remove the role of WCA Delegate from Trainee Delegates in their attributed area.
    7. Senior Delegates can demote Full Delegates to Junior Delegates in their attributed area with approval from the WCA Board.
-   8. Senior Delegates can demote Regional Delegates to Full Delegates in their attributed area with approval from the WCA Board.
+   8. Senior Delegates can demote Regional Delegates to Full Delegates in their attributed area.
```

Comment:
There is this change here where Senior Delegates can demote Regionals without Board approval, yet the updates to the Senior Delegates motion say they do need Board approval to demote.